x-pack/metricbeat/module/aws/sqs/_meta/docs.asciidoc

```diff
@@ -0,0 +1,12 @@
+The sqs metricset of aws module allows you to monitor your AWS SQS queues. `sqs` metricset fetches a set of values from
+https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-available-cloudwatch-metrics.html[Amazon SQS Metrics].
+CloudWatch metrics for Amazon SQS queues are automatically collected and pushed to CloudWatch every five minutes.
```

Comment:
automatically == no user actions needed?

Reply:
Yes, as long as users are using SQS, then these metrics will be collected by cloudwatch with no  additional user actions.